### ISSUE TITLE
TImeZone dropdown clips through screen in Mobile viewport

### ISSUE THREAD

**Issue by harshit078** [twentyhq/twenty#7934]
## Bug Description

- In Settings/Appearance, the timezone dropdown renders the desktop width and clips through screen.
- The dropdown is too transparent and makes it unviewable sometimes
- Filters dropdown doesnt take full width for desktop

## Current Behaviour

<img width="387" alt="Screenshot 2024-10-21 at 6 26 21 PM" src="https://github.com/user-attachments/assets/64ac776a-c769-4bcb-9329-6d2da2040c8e">

<img width="653" alt="Screenshot 2024-10-21 at 6 27 48 PM" src="https://github.com/user-attachments/assets/bcd32a4b-b4d1-4e6a-8b15-39ff8ccc4d5f">
<img width="349" alt="Screenshot 2024-10-21 at 6 31 40 PM" src="https://github.com/user-attachments/assets/989ad1ca-013f-4485-87e4-ad80f0be1fd8">

## Expected behavior
<img width="372" alt="Screenshot 2024-10-21 at 6 29 53 PM" src="https://github.com/user-attachments/assets/25ca98bc-38a1-4a05-8cf7-11e2a977ae11">

<img width="583" alt="Screenshot 2024-10-21 at 6 28 43 PM" src="https://github.com/user-attachments/assets/7463edfa-14cc-4216-93d1-4347d36ecbd6">

**Comment by Bonapara** [comment]
Thanks @harshit078! How did you make the dark mode glitch appear?

**Comment by harshit078** [comment]
@Bonapara  this behaviour appears in mobile viewports not for desktop as there is room for the dropdown to expand below

**Comment by Bonapara** [comment]
Ohh got it! But I don't have a great fix idea for this one.. Do you?

**Comment by Bonapara** [comment]
@harshit078, does this change only concern this dropdown?

**Comment by harshit078** [comment]
@Bonapara Yes, only for this particular dropdown. Rest all dropdowns are optimised

**Comment by martmull** [comment]
![image](https://github.com/user-attachments/assets/158e1037-0b8e-4465-ab43-c820ca2335d6)
It looks that drowdown width matching Select component does not work for all mobile screen widths. Am i wrong with smtg?
